File: library/src/main/java/com/getstream/sdk/chat/utils/strings/ChatStringsImpl.java
package com.getstream.sdk.chat.utils.strings;

import android.content.Context;

public class ChatStringsImpl implements ChatStrings {

    private final Context appContext;

    public ChatStringsImpl(Context appContext) {

        this.appContext = appContext;
    }

    @Override
    public String get(int resId) {
        return appContext.getString(resId);
    }

    @Override
    public String get(int resId, Object... formatArgs) {
        return appContext.getString(resId, formatArgs);
    }
}

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/strings/ChatStringsImpl.kt
package com.getstream.sdk.chat.utils.strings

import android.content.Context

internal class ChatStringsImpl(private val appContext: Context) : ChatStrings {
    override fun get(resId: Int): String {
        return appContext.getString(resId)
    }

    override fun get(resId: Int, vararg formatArgs: Any?): String {
        return appContext.getString(resId, *formatArgs)
    }
}
